vulcan/formula.py

```diff
@@ -75,14 +77,18 @@ def install(self, force=False):
                 shutil.rmtree(destination)
 
             # Move formula into place.
-            origin = os.path.join(tmpdir, self.origin_name)
-            shutil.move(origin, destination)
+            log.info('Moving formula %s to %s' %(self.name, destination))
+            shutil.move(tmpdir, destination)
```

Comment:
Ah, I expect it was due to this:
```yaml
formulas:
  - name: salt-formula
    url: https://github.com/saltstack-formulas/salt-formula.git
    branch: master
    install_directory: formulas
```
the -name is salt-formula which doesn't match because its expecting salt-formula/salt-formula instead of salt-formula/salt - makes sense.